resources/measures/upgrade_hvac_vrf_hr_doas/measure.rb

```diff
@@ -77,6 +77,13 @@ def arguments(model)
     disable_defrost.setDefaultValue(false)
     args << disable_defrost
 
+    # upsizing allowance
+    upsizing_allowance_pct = OpenStudio::Measure::OSArgument.makeDoubleArgument('upsizing_allowance_pct', true)
+    upsizing_allowance_pct.setDisplayName('Upsizing allowance (in %) from cooling design load for heating dominant buildings')
+    upsizing_allowance_pct.setDescription('25% upsizing allowance is the same as 125% from the original size. Setting this value to zero means not applying upsizing.')
+    upsizing_allowance_pct.setDefaultValue(25.0)
```

Comment:
Consider setting the default to 0. We can add the 25% as an option in options lookup.